src/EnergyPlus/PlantManager.cc

```diff
@@ -4531,6 +4541,20 @@ namespace PlantManager {
 		}
 
 	}
+	void
+	CheckOngoingPlantWarnings() 
+	{
+		int LoopNum;
```

Comment:
This variable should be defined in the for loop init statement.  Or better yet...since this isn't an order dependent loop, you could do range based:

```
for ( auto & plantLoop : PlantLoop ) {
  if ( plantLoop.LoopSide...blahblahblha...
```